mockbroker_test.go

```diff
@@ -2,38 +2,63 @@ package sarama
 
 import (
 	"encoding/binary"
-	"errors"
 	"io"
 	"net"
 	"strconv"
+	"sync"
 	"testing"
 	"time"
+
+	"bytes"
+	"fmt"
+	"github.com/davecgh/go-spew/spew"
```

Comment:
is there a particular reason for the import groupings here? typically the 'third-party' imports end up separate from the std lib